app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,191 @@
+$theme-colors: ( "primary": #fe8301);
+$primary-color:#fe8301;
+$body-bg: white;
+$secondary-color:#f2f2f2;
+$font: "Raleway", sans-serif;
+@media screen and (max-width: 1366px) {
+    .navbar-default{
+        background-color: $body-bg;
+    }
+}
+.container{
+  .navbar-brand{
+    img{
+      max-width: 40px;
+       margin-top: -7px;
+    }
+  }
+
+}
+.image-wrapper {
+    margin-top: 50px;
+    margin-bottom: 10px;
+}
+
+.image-hover:hover {
+    transform: scale(1.25);
+    z-index: 2;
+    .col-lg-3:not(:hover) {
+        opacity: 0.2;
+    }
+}
+
+.image-hover {
+    transform: scale(1);
+    transition: 0.3s ease;
+}
+
+nav.navbar {
+    padding-top: 0;
+    margin-bottom: 10px;
+    margin-top: 20px;
+    margin-right: 120px;
+    margin-left: 120px;
+}
+
+.navbar-custom {
+    .navbar-brand {
+        font-size: 12px;
+    }
+}
+
+h1,
+.navbar-brand,
+.navbar-nav a {
+    font-family: $font;
+    color: white;
+    font-size: 12px;
+    margin-right: 10px;
+    margin-left: 10px;
+    font-weight: 900;
+}
+
+#jumbotron-costum {
+    background-image: url("restaurante-cluj.jpg");
+    color: white;
+    font-family: $font;
+    margin-bottom: 0px;
+    min-height: 600px;
+    background-repeat: no-repeat;
+    background-position: center;
+    -webkit-background-size: cover;
+    background-size: cover;
+}
+
+.jumbotron {
+    h1 {
+        text-align: center;
+        padding-top: 60px;
+        font-weight: bolder;
+        font-size: 50px;
+    }
+    p {
+        margin-top: 20px;
+        text-align: center;
+        font-weight: 700px;
+        margin-top: 50px;
+    }
+}
+
+.navbar-default {
+    transition: 500ms ease;
+    background: transparent;
+}
+
+.navbar-default.scrolled {
+    background-color: $secondary-color;
+    margin: 0 0 0 0;
+    padding-top: 10px;
+    border: 2px solid $secondary-color;
```

Comment:
what is the purpose of having a border the same color as the background color? Is it visible?